fuse/schema.graphql

```diff
@@ -9,14 +9,29 @@ type DailyUBI @entity {
 type Citizen @entity {
   id: ID!
   balance: BigInt!
-  inTransfersCount: BigInt!
-  outTransfersCount: BigInt!
-  inTransfersTotal: BigInt!
-  outTransfersTotal: BigInt!
+  inTransactions: BigInt!
+  inTransactionsClean: BigInt!
+  inTransactionsValue: BigInt!
+  inTransactionsValueClean: BigInt!
+  outTransactions: BigInt!
+  outTransactionsClean: BigInt!
+  outTransactionsValue: BigInt!
+  outTransactionsValueClean: BigInt!
+  totalTransactions: BigInt
+  totalTransactionsClean: BigInt
+  totalTransactionsValue: BigInt
+  totalTransactionsValueClean: BigInt
+  lastClaimed: Int
```

Comment:
why Int?
why are some fields mandatory "!" and others not? i dont think they should be mandatory

Reply:
this I actually the citizen entity and should pushed to a different PR and discussed there
changed lastClaimed to BigInt already
most of the fields are not "!" mandatory already
let's discuss the rest at the 2820 PR which I will open soon